cleanlab/datalab/internal/adapter/imagelab.py

```diff
@@ -115,6 +130,14 @@ def collect_issues_from_imagelab(self, imagelab: "Imagelab", issue_types: List[s
             ~self.issue_summary["issue_type"].isin(overlapping_issues)
         ]
         imagelab_summary_copy = imagelab.issue_summary.copy()
+        imagelab_summary_copy = self.filter_based_on_max_prevalence(
+            imagelab_summary_copy, int(IMAGELAB_ISSUES_MAX_PREVALENCE * len(self.issues))
+        )
+
+        imagelab_summary_copy = self.filter_based_on_max_prevalence(
+            imagelab_summary_copy, int(IMAGELAB_ISSUES_MAX_PREVALENCE * len(self.issues))
+        )
```

Comment:
These lines are duplicated:

```suggestion
        imagelab_summary_copy = self.filter_based_on_max_prevalence(
            imagelab_summary_copy, int(IMAGELAB_ISSUES_MAX_PREVALENCE * len(self.issues))
        )
```